packages/rancher-istio/charts/app-readme.md

```diff
@@ -4,47 +4,32 @@ Our [Istio](https://istio.io/) installer wraps the istioctl binary commands in a
 * **[Kiali](https://kiali.io/)**: Used for graphing traffic flow throughout the mesh
 * **[Jaeger](https://www.jaegertracing.io/)**: A quick start, all-in-one installation used for tracing distributed systemm. This is not production qualified, please refer to jaeger documentation to determine which installation you may need instead.
 
-### Known Issues
+For more information on how to use the feature, refer to our [docs](https://rancher.com/docs/rancher/v2.x/en/istio/v2.5/).
+## Known Issues
 
-**Airgapped Environments**
+#### Airgapped Environments
 If you are using this chart in an airgapped environment, you will not be able to upgrade. This is because the `istioctl` upgrade command reaches out to an external repo and it is not configurable. We are tracking the fix for this issue [here](https://github.com/rancher/rancher/issues/33402)
 
-### Dependencies
-
-**Rancher Monitoring  or other Prometheus installation**
-
-The Prometheus CRDs are required for installing Kiali which is enabled by default. If you do not have Prometheus installed your Istio installation will fail. If you do not plan on using Kiali, set `kiali.enabled=false` to bypass this requirement.
-
-### Customization
-
-**Rancher Monitoring**
-
-The Rancher Monitoring app sets `prometheus.prometheusSpec.ignoreNamespaceSelectors=false` which means all namespaces will be scraped by Prometheus by default. This ensures you can view traffic, metrics and graphs for resources deployed in other namespaces.
-
-To limit scraping to specific namespaces, set `prometheus.prometheusSpec.ignoreNamespaceSelectors=true` and add one of the following configurations to ensure you can continue to view traffic, metrics and graphs for your deployed resources.
+## Deprecations
 
-1. Add a Service Monitor or Pod Monitor in the namespace with the targets you want to scrape.
-1. Add an additionalScrapeConfig to your rancher-monitoring instance to scrape all targets in all namespaces.
+#### v1alpha1 security policies
+As of 1.6, Istio deprecated support for `v1alpha1` security policies resource and replaced the API with `v1beta1` authorization policies. https://istio.io/latest/docs/reference/config/security/authorization-policy/
```

Comment:
Isn't it not deprecated but actually removed ? I thought deprecation happened earlier.

Reply:
you are correct - removed not deprecated - I have updated the wording in the latest commits